python/pylibcudf/pyproject.toml

```diff
@@ -97,7 +97,8 @@ skip = [
 ]
 
 [tool.pytest.ini_options]
-addopts = "--tb=native --strict-config --strict-markers"
+# --import-mode=importlib because two test_json.py exists and tests directory is not a structured module
+addopts = "--tb=native --strict-config --strict-markers --import-mode=importlib"
```

Comment:
According to https://docs.pytest.org/en/7.1.x/explanation/pythonpath.html, `--import-mode=importlib` was added in `pytest` 6.0.

So we probably need a floor here:

https://github.com/rapidsai/cudf/blob/7173b52fce25937bb69e22a083a5de4655078fa1/dependencies.yaml#L749

like:

```text
- pytest>=6.0,<8
```

Reply:
This is technically true, but I think we have other reasons why we wouldn't be compatible with lower pytest versions either and we're just generally underspecified. I don't think that we need to address this here and should instead look into this as part of https://github.com/rapidsai/build-planning/issues/105.